lib/core/client.js

```diff
@@ -627,15 +597,15 @@ class Parser extends HTTPParser {
     return request.upgrade ? 2 : request.method === 'HEAD' || statusCode < 200 ? 1 : 0
   }
 
-  [HTTPParser.kOnBody] (chunk, offset, length) {
+  [HTTPParser.kOnBody] (chunk) {
```

Comment:
These changes are no longer necessary

Reply:
Maybe we could consider change it in the future? We can avoid the `chunk.slice` here and maybe increase some throughput?

Reply:
Agreed. Let's land this PR as cleanly as possible and then apply optimizations?